src/routes/+layout.svelte

```diff
@@ -17,6 +18,10 @@
       mediaList.addEventListener('change', changeTheme);
       return () => mediaList.removeEventListener('change', changeTheme);
     }
+    
+    // prefetch used here because of the load time of events query, see #530
```

Comment:
```suggestion
    // prefetch used here to improve the load time of events query, see #530
```

Reply:
Technically not improving load time of query, but of the events page. So should say "refetch used here to improve the load time of the events page, see #530"